src/Tests/Microsoft.NET.TestFramework/Commands/ResolveSdkCommand.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.NET.TestFramework.Commands
+{
+    public sealed class ResolveSdkCommand : MSBuildCommand
```

Comment:
Do we need a separate command to set up the resolver?

Or should all our tests be hooked up to use the resolvers we just built?  In that case we'd set the MSBUILDADDITIONALSDKRESOLVERSFOLDER environment variable in `ToolsetInfo.AddTestEnvironmentVariables` and in the dogfood script.

Reply:
When we hook the build resolvers in they get bypassed by [`DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR`](https://github.com/dotnet/sdk/blob/master/src/Tests/Microsoft.NET.TestFramework/ToolsetInfo.cs#L150). Is that environment variable important or should it be removed in favor of using the newly build resolvers?

Reply:
I think `DOTNET_MSBUILD_SDK_RESOLVER_SDKS_DIR` should not be necessary anymore if we are using the SDK Resolvers correctly.  I would try deleting that line.  You might also want to debug (or look at a binlog for) a test running on full framework to make sure it's resolving the SDK files from the right place.